android/src/main/java/jp/manse/BrightcovePlayerView.java

```diff
@@ -76,7 +76,7 @@ public BrightcovePlayerView(ThemedReactContext context, ReactApplicationContext
         this.applicationContext.addLifecycleEventListener(this);
         this.setBackgroundColor(Color.BLACK);
 
-        this.playerVideoView = new BrightcoveExoPlayerTextureVideoView(context.getCurrentActivity());
+        this.playerVideoView = new BrightcoveExoPlayerTextureVideoView(this.context);
```

Comment:
This reverts a fix for an issue I was seeing in the wild on some android 9 devices, was there a reason you wanted to change it back?

Reply:
No, must have been a merge conflict. We can revert this.